features/auth/useAuthStore.ts

```diff
@@ -118,21 +128,21 @@ export default function useAuthStore() {
         }
         setLoading(false);
       },
-      async updateSignupState(state: SignupFlowRes.AsObject) {
+      async updateSignupState(signupFlowRes: SignupFlowRes) {
+        const state = {
+          flowToken: signupFlowRes.flow_token,
+          isCompleted: !!signupFlowRes.user_created,
+          needAccount: !signupFlowRes.account_is_filled,
+          needFeedback: !signupFlowRes.filled_feedback,
+          needAcceptCommunityGuidelines:
+            !signupFlowRes.accepted_current_community_guidelines,
+        };
         setFlowState(state);
-        if (state.authRes) {
+        if (state.isCompleted) {
           setFlowState(null);
-          authActions.firstLogin(state.authRes!);
           return;
         }
       },
-      async firstLogin(res: AuthRes.AsObject) {
-        setError(null);
-        setUserId(res.userId);
-        Sentry.setUser({ id: res.userId.toString() });
-        setJailed(res.jailed);
-        setAuthenticated(true);
-      },
```

Comment:
Why was this deleted?

Reply:
I guess because the account signup doesn't log the user in now there's no way to use this